modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -735,6 +735,34 @@
 				[b]Note:[/b] As annotations describe their subject, the [annotation @tool] annotation must be placed before the class definition and inheritance.
 			</description>
 		</annotation>
+		<annotation name="@tool_button">
+			<return type="void" />
+			<param index="0" name="text" type="String" default="&quot;&quot;" />
+			<param index="1" name="icon" type="StringName" default="&quot;&quot;" />
+			<description>
+				Mark a function to be displayed as a clickable button in the inspector when running the editor. The parameters are optional and can be used to customize the button.
```

Comment:
There's generally no need to specify when parameters are optional, the signature and the rest of the description should make it clear. "Inspector" should be capitalized.
```suggestion
				Mark a function to be displayed as a clickable button in the Inspector when running the editor.
```